keyboards/noxary/268_2/keymaps/default/keymap.c

```diff
@@ -0,0 +1,83 @@
+#include "n268_2.h"
```

Comment:
Could you replace this:
```suggestion
#include QMK_KEYBOARD_H
```